apps/docs/src/content/reference/extras/orbit-controls.mdx

```diff
@@ -18,6 +18,8 @@
 The component `<OrbitControls>` must be a direct child of a camera component and will mount itself to that camera.
```

Comment:
Maybe in the first line of this page, we should explain about the polar behavior which distinguished `<OrbitControls>` from `<TrackballControls>`. Then, we follow with a "see also..." sentence linking to `<TrackbacllControls>`. Marketing!

Reply:
done!